.github/workflows/browser-tests.yml

```diff
@@ -0,0 +1,48 @@
+name: Browser Tests
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+  schedule:
+    # Every weekday (Mon-Fri) at 7 UTC in the morning
+    - cron: '0 7 * * 1-5'
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Run browser tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
the other project had windows here also? how does it work if we would also have it?

Reply:
Let's try. Perhaps it's good idea in that sense that most end users have windows chrome.

Reply:
windows-latest worked